examples/index.json

```diff
@@ -969,7 +949,7 @@
         "column": 3,
         "category": "Working over the network",
         "samples": [{
-            "name": "NATS Basic Example",
+            "name": "Basic Publisher Subscriber",
```

Comment:
Should this be "Basic Publisher and Subscriber" or "Basic Publish-Subscribe"?

Reply:
"Basic Publisher and Subscriber" more suitable since we have two different files IMO.